theano/tensor/tests/test_opt.py

```diff
@@ -6123,6 +6123,24 @@ def test3(self):
         assert len(divs) == 0
 
 
+def test_local_sumsqr2dot():
+    G = matrix('G')
+    W = matrix('W')
+
+    y = T.sqr( W.dimshuffle('x',0,1) * G.dimshuffle(0,'x',1) ).sum(axis=(1,2))
+
+    f = function([W, G], y)
```

Comment:
use something similar, this should fix travis:

   MODE = theano.compile.get_default_mode().including('local_expm1')